### PR TITLE
getting flags from request body

### PR DESCRIPTION
**Pull request by wheresrhys**
@ironsidevsquincy

### CODE REVIEW

File: server/routes/query.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ import graphql from '../lib/graphql';
 import { HttpError } from '../lib/errors';
 
 export default (req, res) => {
-	const flags = res.locals.flags;
 	const query = req.body.query || req.query.query || req.body;
+	const flags = req.body.flags || {};
```
Comment (ironsidevsquincy) [inline server/routes/query.js]: might still want to use flags when running the app on its own

`Object.assign({}, res.locals.flags, req.body.flags || {})`

**Comment by ironsidevsquincy** [comment]
👍

File: server/routes/query.js (hunk 1 above)
Comment (wheresrhys) [inline server/routes/query.js]: or what I'd prefer (and forgot to do) is disable the flags middleware for this app (paul suspects it's the cause of a lot of memory leaks)

Ideally I think the mockness of the request should be requested via a header I think, but felt like too big a refactor

Comment (ironsidevsquincy) [inline server/routes/query.js]: ok. how would we test the 'mockness' locally without flags?

**Pull request merged**